unittests/finality_proof.hpp

```diff
@@ -247,7 +255,7 @@ namespace finality_proof {
          qc_data_t qc_data = extract_qc_data(block);
 
          // return relevant IBC information
-         return {block, qc_data, onblock_trace, finality_data, active_finalizer_policy.generation, last_pending_finalizer_policy.generation, last_proposed_finalizer_policy.generation, action_mroot, base_digest, active_finalizer_policy_digest, last_pending_finalizer_policy_digest, last_proposed_finalizer_policy_digest, finality_digest, level_3_commitments_digest, level_2_commitments_digest, finality_leaf, finality_root };

```

Comment:
to make it more error-prone, can we use the syntax with the field names (because many members are the same type `digest_type`, if we add a digest_type in the middle the return would still work but be incorrect.

Reply:
Fixed.

Reply:
Nice!